src/services/API.ts

```diff
@@ -96,7 +98,17 @@ export class API {
       `/wallets/${walletAddress}/relay`,
       body
     )
-    return response.data?.safeTxHash
+    return {
+      safeTxHash: response.data.safeTxHash,
+      relayId: response.data.relayId
+    }
+  }
+
+  async getRelayedTransaction(
+    relayId: string
+  ): Promise<RelayedTransactionDetails> {
+    const response = await this.api.get(`/relayed-transactions/${relayId}`)
+    return response.data
```

Comment:
There is not success param in the API return call? We have it for all other API routes, is there a reason your only return the relayed tx with a 200? (I'm not against it, just curious)